requirements.txt

```diff
@@ -11,4 +11,12 @@ openpyxl
 pandas
 python-dateutil
 
```

Comment:
We could add some separating comments on the file so the generated page gets description of the requirements

Reply:
description added